R/dev-example.r

```diff
@@ -33,22 +33,31 @@ dev_example <- function(topic, quiet = FALSE) {
 #' @param run if `TRUE`, code in \code{\\dontrun{}} will be commented
 #'   out.
 #' @param env Environment in which code will be run.
+#' @param macros Custom macros to use to parse the `.Rd` file. See the
+#'   `macros` argument of [tools::parse_Rd()]. If `NULL`, then the
+#'   [tools::Rd2ex()] (and [tools::parse_Rd()]) default is used.
 run_example <- function(path, test = FALSE, run = FALSE,
                         env = new.env(parent = globalenv()),
-                        quiet = FALSE) {
+                        quiet = FALSE, macros = NULL) {
 
   if (!file.exists(path)) {
     stop("'", path, "' does not exist", call. = FALSE)
   }
 
   tmp <- tempfile(fileext = ".R")
 
-  if (getRversion() < "3.2") {
-    # R 3.1 and earlier did not have commentDonttest
-    tools::Rd2ex(path, out = tmp, commentDontrun = !run)
-  } else {
-    tools::Rd2ex(path, out = tmp, commentDontrun = !run, commentDonttest = !test)
+  args <- list(path, out = tmp, commentDontrun = !run)
+
+  # R 3.1 and earlier did not have commentDonttest
```

Comment:
I guess we can remove this conditional now, since we only support 3.2+

Reply:
But we don't necessarily need to _break_ pkgload on 3.1....

Reply:
OK, broke it. :)